crates/nu-cmd-lang/src/core_commands/for_.rs

```diff
@@ -162,7 +162,8 @@ impl Command for For {
                             return Err(err);
                         }
                         Ok(pipeline) => {
-                            let exit_code = pipeline.print(&engine_state, stack, false, false)?;
+                            let exit_code =
+                                pipeline.print_not_formatted(&engine_state, false, false)?;
```

Comment:
What are the implications of this change? Will this switch from using `table` to something else or is this concerned with the display representation of `Value`?

Reply:
@sholderbach - long-term I think we don't want loops to print anything out at all. That'd require the user to use a `print` statement if they wanted output.

This just keeps things mostly the same but at least uses cheaper printing.

Reply:
OK so a breaking change for holdouts at the moment?

Reply:
yeah, hopefully people will already be updating their loops to explicitly call `print`. If they don't, they'll likely break in the future.